english/ghost-buster/ghost-busted.md

```diff
@@ -1,6 +1,6 @@
-After reciting your magic spell to open the door, you then realize you are not Hermione Granger, 
-in fact you are no wizard at all. Upon realizing this demoralizing truth you take a moment to ponder your situation,
-"Alas! I may not be a wizard but I ain't afraid of no ghost nor a marshmallow!" you say aloud after having the epiphany 
+After reciting your magic spell to open the door, you then realize you are not Hermione Granger; 
+in fact you are no wizard at all. Upon realizing this demoralizing truth, you take a moment to ponder your situation.
+"Alas! I may not be a wizard, but I ain't afraid of no ghost, nor a marshmallow!" you say aloud after having the epiphany 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.